README.md

```diff
@@ -1 +1,112 @@
-# Room Access rules
\ No newline at end of file
+# Room Access rules
+
+This module implements handling around the `im.vector.room.access_rules` state event, which spec is described below.
+
+## `im.vector.room.access_rules`
+
+Restricts the access to a room based on the selected preset. Body:
+
+```json
+{
+    "rule": "<rule>"
+}
+```
+
+`<rule>` is either `restricted`, `unrestricted` or `direct`.
+
+The implementation of the different presets lives in the
+`synapse.third_party_rules.access_rules` module.
+
+### `restricted` preset
+
+Default preset for non-direct rooms.
+
+Forbids any invite and membership update for users that belong to a server
+that is in the blacklist provided by the server's configuration
+(`domains_forbidden_when_restricted`). If the invite is a 3PID invite, queries
+a custom `/info` endpoint of the configured identity server to check if that email
+address would belong to a blacklisted server.
```

Comment:
Presumably the invite is then denied if the email address does indeed belong to a blacklisted server? Or is this information simply stored?

Reply:
It's denied. I thought it was implied but I can make it explicit.

Reply:
Huh, I thought I had resolved this thread since I fixed that point.